tests/test_app/Plugin/TestPlugin/config/routes.php

```diff
@@ -1,13 +1,14 @@
 <?php
 
 use Cake\Core\Configure;
+use Cake\Routing\RouteBuilder;
 
 Configure::write('PluginTest.test_plugin.routes', 'loaded plugin routes');
 
-if (isset($routes)) {
+return function (RouteBuilder $routes) {
```

Comment:
I guess this makes it clearer where the `$routes` variable is coming from.

Reply:
Yes, more easier to understand the context.

Reply:
Do we want to throw a deprecation error if a closure is not returned. It might be quite annoying for users hence I am wary of doing do.